api/src/opentrons/drivers/rpi_drivers/types.py

```diff
@@ -12,16 +12,30 @@ class PinDir(enum.Enum):
     output = enum.auto()
 
 
+class PortGroup(enum.Enum):
+    MAIN = enum.auto()
+    LEFT = enum.auto()
```

Comment:
Can we make these have string values? So-

```suggestion
    MAIN = "main"
    LEFT = "left"
```
This way, the robot server's responses (which currently would show `portGroup` as a number) would show the string value instead, which is a more helpful response.